### PR TITLE
feat: added new badge model

### PR DESCRIPTION
**Pull request by allisonsuarez**
### Summary of Changes

Added a model in the table file for Badge.
### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes.

### CODE REVIEW

File: amundsen_common/models/table.py [hunk 1]
```diff
@@ -33,6 +33,19 @@ class Meta:
         register_as_scheme = True
 
 
+@attr.s(auto_attribs=True, kw_only=True)
+class Badge:
+    badge_name: str = attr.ib()
+    badge_type: str = attr.ib()
+    sentiment: str = attr.ib()
```
Comment (feng-tao) [inline amundsen_common/models/table.py:40]: let's still call it category which could be more generic for other use cases.

Comment (feng-tao) [reply amundsen_common/models/table.py:40]: do you plan to `column`, `table status` in category and negative / neutral in badge_type?

**Comment by feng-tao** [comment]
we also need a version bump for common repo.

File: amundsen_common/models/table.py (hunk 1 above)
Comment (allisonsuarez) [reply amundsen_common/models/table.py:40]: column, table status in category and negative / neutral in sentiment

Comment (feng-tao) [reply amundsen_common/models/table.py:40]: I thought we are going with badge_type and category?

Comment (allisonsuarez) [reply amundsen_common/models/table.py:40]: I thought we had said that sentiment was a good descriptor for the color of the badge (positive, negative, neutral), while category would be specific to the type of badge, that's why I renamed category to be badge_type.

**Review by feng-tao — DISMISSED** [review]

File: setup.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='amundsen-common',
-    version='0.3.6',
+    version='0.4.6',
```
Comment (ttannis) [inline setup.py:8]: If you're bumping the minor version then the patch version restarts at 0, so `0.4.0`

**Pull request merged**